forage-android/src/main/java/com/joinforage/forage/android/network/model/Payment.kt

```diff
@@ -0,0 +1,158 @@
+package com.joinforage.forage.android.network.model
+
+import com.joinforage.forage.android.getStringOrNull
+import com.joinforage.forage.android.hasNonNull
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * @property city The name of the city.
+ * @property country Either us or US. Defaults to US if not provided.
+ * @property state The two-letter abbreviation, can be upper or lowercase, for the US state.
+ * @property zipcode The zip or postal code.
+ * @property line1 The first line of the street address.
+ * @property line2 The second line of the street address.
+ */
+data class Address(
+    val city: String,
+    val country: String,
+    val state: String,
+    val zipcode: String,
+    val line1: String?,
+    val line2: String?
+) {
+    internal constructor(jsonString: String) : this(JSONObject(jsonString))
+    internal constructor(jsonObject: JSONObject) : this(
+        city = jsonObject.getString("city"),
+        country = jsonObject.getString("country"),
+        zipcode = jsonObject.getString("zipcode"),
+        state = jsonObject.getString("state"),
+        line1 = jsonObject.getStringOrNull("line1"),
+        line2 = jsonObject.getStringOrNull("line2")
+    )
+}
+
+/**
+ * @property created A UTC-8 timestamp of when the Receipt was created, represented as an ISO 8601 date-time string.
+ * @property ebtCashAmount The USD amount charged/refunded to the EBT Cash balance of the EBT Card, represented as a numeric string.
+ * @property isVoided A boolean that indicates whether the Receipt is voided.
+ * @property last4 The last four digits of the EBT Card number that was charged.
+ * @property message A message from the EBT payment network that must be displayed to the EBT cardholder.
+ * @property salesTaxApplied The USD amount of taxes charged to the customer’s non-EBT payment instrument, represented as a numeric string.
+ * @property snapAmount The USD amount charged/refunded to the SNAP balance of the EBT Card, represented as a numeric string.
+ * @property balance The remaining balance on the EBT Card after the Payment was processed.
+ */
+data class Receipt(
+    val created: String,
+    val ebtCashAmount: String,
+    val isVoided: Boolean,
+    val last4: String,
+    val message: String,
+    val otherAmount: String,
+    val salesTaxApplied: String,
```

Comment:
Those fields appear on the object itself, but yes they are unused for standalone Payment objects. The values will always be 0. The choice is:
1. Keep them for consistency sake across other platforms and our docs
2. Remove them because they will always be unused

I lean towards removing them personally.

Reply:
I lean towards removing as well!

Reply:
Done! ea89a177e2daa74b3af03d720a96f31dee978ec0